lib/resources/example_json/r4_examples_care_plan.rb

```diff
@@ -124,5 +186,88 @@ module Resources
         }
       ]
     }.freeze
+
+    R4_CARE_PLAN_PATIENT_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': '013ed260-cdd4-4ffc-947e-aeadb699d3d4',
+      'type': 'searchset',
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/CarePlan?patient=12724066&category=assess-plan'
+        }
+      ],
+      'entry': [
+        {
+          'fullUrl': 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/CarePlan/197375407',
+          'resource': R4_CARE_PLAN_PATIENT_ENTRY
+        },
+        {
+          'fullUrl': 'https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/CarePlan/197286315',
+          'resource': {
+            'resourceType': 'CarePlan',
+            'id': '197286315',
+            'meta': {
+              'versionId': '1',
+              'lastUpdated': '2020-04-05T04:00:46.000Z'
+            },
+            'text': {
+              'status': 'additional',
+              'div': '&lt;div xmlns="http://www.w3.org/1999/xhtml"&gt;&lt;p&gt;&lt;b&gt;'\
```

Comment:
Same as above. Update div tags instead of &lt;

Reply:
Fixed